client-app/shared/catalog/components/facet-filter.vue

```diff
@@ -106,22 +198,169 @@ const isAnchorAdded = computed(() => searchedValues.value.length > MAX_ITEMS_VIS
 
 const hasFade = computed(
   () =>
-    (searchedValues.value.length > SHOW_MORE_AMOUNT && !isExpanded.value) ||
+    (filtered.value.length > SHOW_MORE_AMOUNT + 1 && !isExpanded.value) ||
     (isAnchorAdded.value && !fadeVisibilityAnchorIsVisible.value),
 );
+const selectedFiltersCount = computed(() => facet.value.values.filter((item) => item.selected)?.length);
+const hasSelected = computed(() => selectedFiltersCount.value > 0);
 </script>
 
-<style scoped lang="scss">
-.fade-bottom {
-  --scrollbar-width: 15px;
+<style lang="scss">
+.facet-filter {
+  $collapsable: "";
+  $dropdown: "";
+  $selectedItem: "";
+  $fadeBottom: "";
+  $triggerOpened: "";
+
+  &--collapsable {
+    $collapsable: &;
+  }
+
+  &--dropdown {
+    $dropdown: &;
+
+    .vc-popover__content {
+      @apply min-w-44;
+    }
+
+    .vc-menu-item__inner {
+      @apply py-1.5 px-4;
+    }
+
+    .vc-button--color--secondary {
+      @apply text-[var(--color-secondary-600)] #{!important};
+    }
+  }
+
+  &--fade-bottom {
+    $fadeBottom: &;
+  }
+
+  .vc-checkbox__label {
+    @apply w-full;
+  }
+
+  &__trigger {
+    @apply border-2 border-r-4;
+
+    &--opened {
+      $triggerOpened: &;
+    }
+  }
+
+  &__wrapper {
+    @apply relative z-10;
+  }
+
+  &__trigger-wrapper {
+    @apply flex items-center gap-2;
+  }
+
+  &__trigger-arrow {
+    @apply ml-2 transition-transform;
+
+    #{$triggerOpened} & {
+      @apply rotate-180;
+    }
+  }
+
+  &__item-input {
+    @apply w-full;
+  }
+
+  &__item {
+    &--selected {
+      $selectedItem: &;
+    }
+  }
+
+  &__item-inner {
+    @apply flex items-center;
+  }
+
+  &__item-count {
+    @apply ml-auto;
+  }
+
+  &__content {
+    #{$dropdown} & {
+      @apply max-w-72 overflow-y-auto py-2;
+    }
+
+    #{$fadeBottom}#{$dropdown} & {
+      @apply pb-0;
+    }
+
+    #{$fadeBottom} & {
+      --scrollbar-width: 15px;
+      @apply relative;
+
+      &:after {
+        @apply w-full absolute block bottom-0 content-[''] h-10 bg-gradient-to-t from-white;
+
+        @media print {
+          @apply content-none;
+        }
+      }
+    }
+  }
+
+  &__search {
+    #{$collapsable} & {
+      @apply mb-4;
+    }
+
+    #{$dropdown} & {
+      @apply mx-4 my-2;
+    }
+  }
 
-  position: relative;
-  &:after {
-    width: calc(100% - var(--scrollbar-width));
-    @apply absolute block bottom-0 content-[''] h-10 bg-gradient-to-t from-white;
+  &__items-container {
+    #{$collapsable} & {
+      @apply -me-1 space-y-3 overflow-y-auto pe-1;
+
+      max-height: v-bind(maxHeight);
+    }
+  }
+
+  &__item-label {
+    @apply truncate;
+
+    #{$dropdown} & {
+      @apply mr-5 text-13 font-medium text-gray-500;
```

Comment:
text-13 - deprecated
font lato doesn’t support **font-medium**, **font-semibold**, **font-extrabold** (see font lato documentation)
text-gray-500 - we have to use colors from palette only

⬆️  fix it everywhere